python/cudf/cudf/_lib/pylibcudf/strings/CMakeLists.txt

```
@@ -12,10 +12,10 @@
 # the License.
 # =============================================================================
 
-set(cython_sources case.pyx find.pyx)
+set(cython_sources case.pyx find.pyx replace.pyx)
 set(linked_libraries cudf::cudf)
 rapids_cython_create_modules(
   CXX
   SOURCE_FILES "${cython_sources}"
-  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX pylibcudf_ ASSOCIATED_TARGETS cudf
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX pylibcudf_strings ASSOCIATED_TARGETS cudf
```

Good idea to rename, but let's do
```suggestion
  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX pylibcudf_strings_ ASSOCIATED_TARGETS cudf
```